src/css/jwplayer/states.less

```diff
@@ -4,6 +4,10 @@
 .jw-state-setup {
     background-color: transparent;
 
+    .jw-wrapper {
+        background-color: transparent;
```

Comment:
Should it always be transparent? (not just on setup). Please compare the current Views page in this build (http://player-pr-test-jenkins.longtailvideo.com/builds/17782/archive/test/view/) with an earlier version before the wrapper was added.

Reply:
The wrapper is used for floating player, and it needs the black background for floating player.
Without the background, there is no black space when there needs to be.

Reply:
But it is also present when we're not in floating mode, so the black background is applied everywhere except for setup. 

`.jw-wrapper { background-color: transparent; } ` should be a default (in jwplayerlayout not here in state-setup) setting and only make it black when floating.